src/nvim/move.c

```diff
@@ -1020,16 +1020,12 @@ void textpos2screenpos(win_T *wp, pos_T *pos, int *rowp, int *scolp,
   *ecolp = ecol + coloff;
 }
 
-/*
- * Scroll the current window down by "line_count" logical lines.  "CTRL-Y"
- */
-void 
-scrolldown (
-    long line_count,
-    int byfold              /* true: count a closed fold as one line */
-)
+/// Scroll the current window down by "line_count" logical lines.  "CTRL-Y"
```

Comment:
a blank `///` line is typical after the first one.

Reply:
Totally right, saw the rest but totally forgot to match it. It's done.